src/App.scss

```diff
@@ -1,13 +1,13 @@
-$primary: #14213D;
+$primary: #ADA0A6;
```

Comment:
does this change do anything? or does it just reflect what we want these colors to be right now 😅

Reply:
lol it doesnt work, and yea they're just the desired colours